File: İlişkilendirici(C#)/Assosicate/Assosicate/Form1.vb
Public Class Form1


    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load


        Dim arguments As String = Command()
        If Not arguments = String.Empty Then
            Dim tempstr = arguments.Replace("""", "")
            My.Computer.FileSystem.CopyFile(tempstr, "C:\Projector\Proje\OD\projem.pjx")
            Shell("C:\Projector\Projector\decompress.exe")
            Close()


        End If
    End Sub


End Class

File: pjx ŞİFRE Çözücü/decompressor/decompressor/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.IO.Compression;
using System.Diagnostics;

namespace decompressor
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            ZipFile.ExtractToDirectory(@"C:\Projector\Proje\OD\projem.pjx", @"C:\Projector\Proje\OD\DC");
            Process.Start(@"C:\Projector\Projector\onlyLOADER.exe");
            Close();
            
        }
    }
}
